src/nprintml/cli.py

```diff
@@ -57,7 +58,7 @@ def build_parser(**parser_kwargs):
     """
     parser = argparse.ArgumentParser(
         description='train models for network traffic analysis',
-        formatter_class=argparse_formatter.ParagraphFormatter,
+        formatter_class=argparse_formatter.FlexiFormatter,
```

Comment:
(We can also do more to make the help documentation prettier but it's not bad as is.)